src/Composer/Package/Package.php

```diff
@@ -20,6 +20,8 @@
  * Core package definitions that are needed to resolve dependencies and install packages
  *
  * @author Nils Adermann <naderman@naderman.de>
+ *
+ * @phpstan-import-type AutoloadMapping from \Composer\Package\PackageInterface
```

Comment:
I think this should work too?

```suggestion
 * @phpstan-import-type AutoloadMapping from PackageInterface
```

Reply:
Yes, you are right. Thanks for suggestion!
I would commit this "Suggested change"
 
NOTES: Using FQCN is just by my current habit. Because  I often met other coding standard check tool reports "Unused Use" false-positive. :cry: